cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -5754,7 +5754,32 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = VPN
 action.cyences_notable_event_action.teams = SOC
 
-
+[Authentication - DUO Locked Out]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 4 * * * *
+description = This alert will pinpoint when a user is locked out from Duo app. 
+dispatch.earliest_time = -4h@h
+dispatch.latest_time = +4h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = index="duo" source="duo" _index_earliest=-61m@m _index_latest=-1m@m reason="Locked out" \
+| eval cyences_severity = if(is_privileged_user=="Yes", "critical","high") \
```

Comment:
map the user privilege before using is_privileged_user field.

| `cs_user_privilege_mapping(username)`